container/docker_compose_setup/dockerfiles/MySQL_Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM mysql/mysql-server:5.7
```

Comment:
I think we may rename this file "MySQL57_Dockerfile", and add a second named "MySQL80_Dockerfile" for the MySQL 8.0 version. 
Similar can be said for the "DIRAC_Dockerfile" and slc6 vs cc7.

Reply:
Sure will make these changes.